### PR TITLE
Sprint 12

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: docs/zap-report.html [hunk 1]
```diff
@@ -822,6 +794,20 @@ <h3>Alert Detail</h3>
   
   
   
+<tr bgcolor="#e8e8e8">
+<td width="20%" class="indent1">URL</td><td width="80%">https://efcms-dev.ustc-case-mgmt.flexion.us/v1/users/4513147316483627382</td>
```
Comment (matthopson) [inline docs/zap-report.html:798]: Is this intentionally hard-coded?

File: efcms-service/src/cases/getCasesByUserLambda.js [hunk 2]
```diff
@@ -9,12 +9,20 @@ const createApplicationContext = require('../applicationContext');
...
+  handle(event, async () => {
     const userId = (event.pathParameters || {}).userId;
     const user = getUserFromAuthHeader(event);
     const applicationContext = createApplicationContext(user);
-    return applicationContext.getUseCases().getCasesByUser({
-      applicationContext,
-      userId,
-    });
+    try {
+      const results = applicationContext.getUseCases().getCasesByUser({
```
Comment (matthopson) [inline efcms-service/src/cases/getCasesByUserLambda.js:17]: Should this `await`?

File: efcms-service/src/cases/getCasesForRespondentLambda.js [hunk 3]
```diff
@@ -9,12 +9,20 @@ const createApplicationContext = require('../applicationContext');
...
+  handle(event, async () => {
     const respondentId = (event.pathParameters || {}).respondentId;
     const user = getUserFromAuthHeader(event);
     const applicationContext = createApplicationContext(user);
-    return applicationContext.getUseCases().getCasesForRespondent({
-      applicationContext,
-      respondentId,
-    });
+    try {
+      const results = applicationContext.getUseCases().getCasesForRespondent({
```
Comment (matthopson) [inline efcms-service/src/cases/getCasesForRespondentLambda.js:17]: Same here, `await`?

Comment (matthopson) [reply efcms-service/src/cases/getCasesForRespondentLambda.js:17]: There are a few others - so I'm assuming they're being intentionally left off. Just wanted to bring them up.

**Review by codyseibert — APPROVED** [review]

File: docs/zap-report.html (hunk 1 above)
Comment (ericsorenson) [reply docs/zap-report.html:798]: This is an autogenerated report from zap, maybe they take PRs :-)

File: efcms-service/src/cases/getCasesByUserLambda.js (hunk 2 above)
Comment (ericsorenson) [reply efcms-service/src/cases/getCasesByUserLambda.js:17]: Yes it should, good catch :-)

File: efcms-service/src/cases/getCasesForRespondentLambda.js (hunk 3 above)
Comment (ericsorenson) [reply efcms-service/src/cases/getCasesForRespondentLambda.js:17]: Yes it should :-)

**Review by matthopson — APPROVED** [review]
👍

**Pull request merged**